src/file_management/core.clj

```diff
@@ -1,10 +1,23 @@
 (ns file-management.core
+  (use clojure.java.io)
+  (:import (org.apache.commons.codec.binary Base64))
   (:require [aws.sdk.s3 :as s3]))
 
 (defn build-avatar-key
   [user-id]
   (str user-id "-avatar"))
 
+(defn get-avatar-url
+  [bucket user-id]
+  (str "https://" bucket ".s3.amazonaws.com/" (build-avatar-key user-id)))
```

Comment:
Yep! We can go on incógnito because of this https://github.com/clanhr/file-management/pull/2/files#diff-c2c7b5169cdc594006d308fcb74bbdfaR33